src/main/java/bridge/view/OutputView.java

```diff
@@ -0,0 +1,73 @@
+package bridge.view;
+
+import bridge.model.AttemptCount;
+import bridge.model.BridgeState;
+import bridge.model.MoveResult;
+import bridge.utils.enums.MoveCommend;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static bridge.utils.enums.IOMessage.*;
+
+/**
+ * 사용자에게 게임 진행 상황과 결과를 출력하는 역할을 한다.
+ */
+public class OutputView {
+    private static final String CORRECT = "O";
+    private static final String WRONG = "X";
+    private static final String SPACE = " ";
+    private static final String DIVIDER = " | ";
+    private static final String OPEN = "[ ";
+    private static final String CLOSE = " ]";
+
+    public static void printGameStart() {
+        System.out.println();
+    }
+
+    /**
+     * 현재까지 이동한 다리의 상태를 정해진 형식에 맞춰 출력한다.
+     * <p>
+     * 출력을 위해 필요한 메서드의 인자(parameter)는 자유롭게 추가하거나 변경할 수 있다.
+     */
+    public static void printMap(BridgeState bridgeState) {
+        List<MoveResult> results = bridgeState.getMoveResults();
+        List<String> upBridgeState = new ArrayList<>();
+        List<String> downBridgeState = new ArrayList<>();
+        String mark = WRONG;
+        for (MoveResult result : results) {
+            if(result.isSuccess())
+                mark = CORRECT;
+            if (result.isEqualDirection(MoveCommend.UP)) {
+                upBridgeState.add(mark);
+                downBridgeState.add(SPACE);
+            } else if (result.isEqualDirection(MoveCommend.DOWN)) {
+                downBridgeState.add(mark);
+                upBridgeState.add(SPACE);
+            }
+        }
+        printFormattedBridge(upBridgeState, downBridgeState);
+    }
+
+    private static void printFormattedBridge(List<String> upBridgeState, List<String> downBridgeState) {
+        String upResults = String.join(DIVIDER, upBridgeState);
+        String downResults = String.join(DIVIDER, downBridgeState);
+        System.out.println(OPEN + upResults + CLOSE);
+        System.out.println(OPEN + downResults + CLOSE);
+    }
+
+    /**
+     * 게임의 최종 결과를 정해진 형식에 맞춰 출력한다.
+     * <p>
+     * 출력을 위해 필요한 메서드의 인자(parameter)는 자유롭게 추가하거나 변경할 수 있다.
+     */
+    public static void printResult(BridgeState bridgeState, AttemptCount attemptCount, boolean isSuccess) {
+        String successMessage = OUTPUT_FAIL.getMessage();
+        if (isSuccess)
+            successMessage = OUTPUT_SUCCESS.getMessage();
```

Comment:
> Java 코드 컨벤션에 따르면 4.1.1 괄호는 선택사항에서도 쓰인다.
괄호는 if, else, for, do, while 구문에 쓰이는데 몸체가 없거나 한 줄의 구문에도 괄호가 쓰인다.
https://github.com/JunHoPark93/google-java-styleguide

```suggestion
        if (isSuccess) {
            successMessage = OUPUT_SUCCESS.getMessage();
        }
```

괄호 표기해주는게 좋을 것 같습니다 !